apps/studio/prisma/scripts/migrations/addChildrenBlock/addChildrenBlock.ts

```diff
@@ -0,0 +1,58 @@
+import { DEFAULT_CHILDREN_PAGES_BLOCK } from "@opengovsg/isomer-components"
+
+import { db, sql } from "~/server/modules/database"
+import { updateBlobById } from "~/server/modules/resource/resource.service"
+
+export const up = async () => {
+  // Step 1: select all index pages and their blobs where
+  // the text `"type": "childrenpages"` isn't found
+  const sqlQuery = sql<{
+    id: string
+    siteId: string
+    content: PrismaJson.BlobJsonContent
+  }>`
+    SELECT
+      *
+    FROM
+      (
+        SELECT
+          COALESCE("db"."content", "pb"."content") AS "content",
+          "Resource"."id", 
+          "Resource"."siteId"
+        FROM
+          "Resource"
+          LEFT JOIN "Version" ON "Version"."id" = "Resource"."publishedVersionId"
+          LEFT JOIN "Blob" AS "pb" ON "pb"."id" = "Version"."blobId"
+          LEFT JOIN "Blob" AS "db" ON "db"."id" = "Resource"."draftBlobId"
+        WHERE
+          "type" = 'IndexPage'
+      ) AS "indexContent"
+    WHERE
+      ("content"->'content')::text NOT ILIKE '%"type": "childrenpages"%' and "content" ->> 'layout' = 'index';
```

Comment:
question(non-blocking): ah I read the `addOrdering` file first, so to confirm, we will:
1. Run this `addChildrenBlock` first to add the `childrenpages` block to all index pages, then
2. Run the `addOrdering` script to add the `childrenPagesOrdering`?

Reply:
the order is:

1. run `addOrdering` so that every **existing** index page **with childrenpages block but without ordering** now has ordering in their child block
2. run `addChildrenBlock` so that any **existing index page without childrenpages block** now has the default children block (with `ordering`) 

(together, this means that all index pages have childrenpagse + ordering)